src/python/zquantum/optimizers/cma_es_optimizer.py

```diff
@@ -10,58 +8,49 @@
 )
 from scipy.optimize import OptimizeResult
 import cma
+from typing import Dict, Optional
 
 
 class CMAESOptimizer(Optimizer):
-    def __init__(self, options):
+    def __init__(
+        self, sigma_0: float, options: Optional[Dict] = None, recorder=_recorder
```

Comment:
Typing needed for recorder.

Reply:
Fixed in https://github.com/zapatacomputing/z-quantum-optimizers/pull/58/commits/d4fba8b8bd7930fd68bc5a0388d153201c79bc4b